packages/react-components/src/hooks/useAlarms/utils/alarmDataUtils.ts

```diff
@@ -27,10 +27,18 @@ export const buildFromAssetResponse = ({
   status: AlarmDataStatus;
   assetResponse?: DescribeAssetResponse;
 }): AlarmData[] => {
-  const { assetId, assetCompositeModelId, inputPropertyId } = request;
+  const { assetId, assetCompositeModelId, inputPropertyId, assetModelId } =
+    request;
 
   if (!assetResponse || !assetId) {
-    return [];
+    return [
+      {
+        assetModelId,
```

Comment:
nit: there will never be an assetModelId in the request here cause it would have been filtered out